README.md

```diff
@@ -3,50 +3,41 @@
 # Alertbot
 
 This bot uses the webhook functionality of monitoring/alerting solutions like Grafana or Alertmanager to forward alerts to matrix rooms.
-This means that you no longer have to use E-Mail or Slack to receive alerts. See the setup below for how to use it or
-join the [Alertbot room on matrix](https://matrix.to/#/#alertbot:hyteck.de)
+This means that you no longer have to use E-Mail or Slack to receive alerts. 
 
-# Getting Started
+You can either [invite](#invite-the-official-instance) an official instance of the bot, or [self-host](#self-host-an-instance) it on your own matrix server.
 
-## OPTION 1: Use provided alertbot
+## Invite the Official Instance
 
 * Create a Matrix room and invite @alertbot:hyteck.de
 * Send `!url` to the room. The bot will answer with the webhook URL
 * Put the Webhook URL into your monitoring solution (see below)
 
-## OPTION 2: Selfhost alertbot
+## Self-host an Instance
 
 **Prerequisites:**
-* A Matrix server where you have access to a maubot instance: Please [refer to the docs](https://docs.mau.fi/maubot/usage/setup/index.html) for setting up one
+* A Matrix server where you have access to a maubot instance
+   * Refer to the [docs](https://docs.mau.fi/maubot/usage/setup/index.html) for setting up one
+   * or use [spantaleev/matrix-docker-ansible-deploy](https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/master/docs/configuring-playbook-bot-matrix-registration-bot.md) which has built-in maubot support
 * An instance of alertmanager or grafana or a similar alerting program that is able to send webhooks
 
-**Getting the code**
+**Build the Bot**
 
-Clone this repository to your local computer and install maubot to have access to the maubot CLI
+Clone this repository and build it into a maubot plugin file (.mbp). Navigate to the Maubot Administration Interface and upload the .mpb file.
```

Comment:
Sure, that sounds reasonable. The reason I altered this is because in the matrix-docker-ansible-deploy repository the container is read-only, so build and uploading from inside the container was impossible